nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -151,11 +151,13 @@ in
       };
 
       daemonIONiceLevel = mkOption {
-        type = types.int;
+        type = types.either types.int (types.enum ["idle"]);
```

Comment:
I like that idea, except that it's not backwards-compatible.  How would we deal with that issue?